### ISSUE TITLE
Project Collections grid is not keyboard accessible 

### ISSUE THREAD

**Issue by zdavis** [ManifoldScholar/manifold#2415]
The library we use here is different from our other D&D interfaces. We used a different library because we needed grid drag and drop support. AFAIK, this library is not keyboard accessible, and will likely need to be replaced. I also wonder if a list interface like what we have on the other D&D components isn't a better UI choice anyway.

**Comment by dananjohnson** [comment]
Looks like this library does provide keyboard accessibility. We just need to modify the markup to allow the grid item wrappers to be focusable. We should also hook drag events up to the `withScreenReaderStatus` HOC.

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by Santoz2** [comment]
Accessibility Triage Notes: This is a priority 1 issue, as keyboard users are completely unable to reorder a collection.

**Comment by dananjohnson** [comment]
@zdavis revisiting this issue, I'm inclined toward your suggestion of bringing this interface in line with the other DnD interfaces on the backend. It would be more consistent, and I agree that re-ordering items in a list is probably more intuitive than in a grid. This work could coincide with #2437 (updating `react-beautiful-dnd`).

**Comment by Santoz2** [comment]
4/27/2020 UW ATX Verification: Fixed, Project Collections can now be reordered with a keyboard.
